src/extension.ts

```diff
@@ -88,6 +89,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 	if (vscode.window.registerWebviewPanelSerializer) {
 		// Make sure we register a serializer in activation event
 		vscode.window.registerWebviewPanelSerializer(DidactWebviewPanel.viewType, {
+			// eslint-disable-next-line @typescript-eslint/no-unused-vars
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Is it that we are in the case for "when not to use it"? https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-explicit-any.md#when-not-to-use-it

`If an unknown type or a library without typings is used and you want to be able to specify any.`

Reply:
I think the 'any' vs 'unknown' debate for eslint should probably be debated elsewhere, but I will remove any disabled rules for the PR.